eng/common/scripts/stress-testing/stress-test-deployment-lib.ps1

```diff
@@ -302,7 +302,7 @@ function DeployStressPackage(
             }
         }
         $generatedHelmValues.scenarios = @( foreach ($scenario in $generatedHelmValues.scenarios) {
-            $dockerPath = if ("image" -notin $scenario) {
+            $dockerPath = if ("image" -notin $scenario.keys) {
```

Comment:
`$scenarios` is a dictionary, and `notin` checks if item is in the collection, so it never contained `image` item and did `$scenario.imageTag = $imageTag` for all scenarios on line 311